filebeat/docs/configuration.asciidoc

```diff
@@ -307,6 +307,15 @@ filebeat:
 -------------------------------------------------------------------------------------
 
 
+===== publish_async
+
+Experimental!
```

Comment:
See the sources from here: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-pipeline.html The `experimental[]` string seems to generate a warning. Not sure if that works well in this context, though.